src/main/java/io/anserini/search/SearchCollection.java

```diff
@@ -606,7 +607,31 @@ public void close() throws IOException {
           }
         }
       }
-    } else {
+    } else if (args.rocchio) {
+      for (String fbTerms : args.rocchio_fbTerms) {
+        for (String fbDocs : args.rocchio_fbDocs) {
+          for (String alpha : args.rocchio_alpha) {
+            for (String beta : args.rocchio_beta) {
+              String tag;
+              if (this.args.rf_qrels != null){
+                tag = String.format("rocchioRf(fbTerms=%s,alpha=%s,beta=%s)",
```

Comment:
I'm fine with longish lines... code would look better on a single line.

And below.